README.md

```diff
@@ -295,7 +295,7 @@ The relayer is configured via a JSON file, the path to which is passed in via th
 
 `"decider-port": unsigned integer`
 
-- The network location of a service implementing the gRPC service defined by `proto/decider`, which will be queried for each message to determine whether that message should be relayed. If a port is specified but a host is not, the host is assumed to be `localhost`.
+- The network location of a service implementing the gRPC service defined by `proto/decider`, which will be queried for each message to determine whether that message should be relayed. If a port is specified but a host is not, the host is assumed to be `localhost`. If a host is specified then a port is required.
```

Comment:
I've been thinking about this a bit, and IMO the semantics of a default `decider-host` set to `localhost` is unnecessarily complex. I think it's much easer to reason about if both fields are required to be populated, or both fields are required to be omitted. Thinking about somebody reading a config file, it seems like it would be confusing to see `decider-port` specified without a corresponding `decider-host`.

Reply:
Thanks for insisting. I changed it to use just a URL like you original suggested, in https://github.com/ava-labs/awm-relayer/pull/344/commits/7eb5112b149ca2ed3bf321f1a3a5d7d081c787d7